build-support/bin/generate_github_workflows.py

```diff
@@ -481,30 +466,18 @@ def bootstrap_pants(self) -> Sequence[Step]:
             self.native_binaries_upload(),
         ]
 
-    def build_wheels(self, python_versions: list[str]) -> list[Step]:
+    def build_wheels(self) -> list[Step]:
         cmd = dedent(
             # Note that the build-local-pex run is just for smoke-testing that pex
             # builds work, and it must come *before* the build-wheels runs, since
             # it cleans out `dist/deploy`, which the build-wheels runs populate for
             # later attention by deploy_to_s3.py.
             """\
-            USE_PY39=true ./build-support/bin/release.sh build-local-pex
+            ./pants run build-support/bin/release.py build-local-pex
```

Comment:
The release.sh script only existed to handle setting the python version based on an env var, so we get rid of it.